.github/workflows/googleapis-update.yml

```diff
@@ -0,0 +1,40 @@
+name: Update Google APIs Commit Hash
+
+on:
+  schedule:
+    - cron: '0 0 1 * *'
```

Comment:
only you get pinged when these fail. Probably need to update this again with a group/bot account after submit.

Reply:
once https://github.com/sigstore/community/pull/535 merges I'll add a clause here to open an issue and tag folks

Reply:
This has merged

Reply:
i'll have to add the FYI feature back first.